src/ASM/ASMs3Dmx.C

```diff
@@ -97,7 +97,7 @@ bool ASMs3Dmx::readBasis (std::istream& is, size_t basis)
 bool ASMs3Dmx::write (std::ostream& os, int basis) const
 {
   if (basis == -1)
-    os <<"700 1 0 0\n" << *proj;
+    os <<"700 1 0 0\n" << *static_cast<const Go::SplineVolume*>(projB);
```

Comment:
No need to cast here, since the operator<< is defined in the base class Go::Streamable?

Reply:
indeed.